### ISSUE TITLE
Setting up Travis

### ISSUE THREAD

**Issue by nicola** [linkeddata/ldnode#28]
We need the travis config file

https://travis-ci.org/linkeddata/ldnode/builds/67748220

**Comment by nicola** [comment]
Should be done here https://github.com/linkeddata/ldnode/pull/32

**Comment by martinmr** [comment]
@nicola: Would you like to see if this issue can be resolved now that rdflib is published on npm?

**Comment by martinmr** [comment]
@nicola : Does your latest pull request fix this issue?

**Comment by nicola** [comment]
interesting, travis was passing for my repository.. https://travis-ci.org/nicola/ldnode

**Comment by martinmr** [comment]
Not sure what's going on. I checked on my local machine and the tests on the linkeddata repository are all working.

**Comment by nicola** [comment]
so the difference between the travis on my repo and on this, is that npm fails to parse the json

**Comment by martinmr** [comment]
I validated package.json and it is valid json.

**Comment by nicola** [comment]
Interesting, I will have a look tonight. Yes, Travis now reads the package.json fine now, but still fails the tests

**Comment by martinmr** [comment]
I looked at the error log. The errors are similar to what I was getting in an earlier version of rdflib, which had a bug that's already fixed in the latest release (0.2.6). Maybe you need to force "npm install" to update all the packages.

**Comment by nicola** [comment]
```
module.js:338
    throw err;
    ^
Error: Cannot find module 'underscore'
    at Function.Module._resolveFilename (module.js:336:15)
    at Function.Module._load (module.js:278:25)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at Object.<anonymous> (/home/travis/build/nicola/ldnode/handlers/get.js:4:9)
    at Module._compile (module.js:460:26)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Module.require (module.js:365:17)
make: *** [run-for-test] Error 1
```
